_raml/apps/products/examples/patch_product_productid_response.json

```diff
@@ -83,6 +83,119 @@
     "manufacturerPrice": null,
     "basePrice": null
   },
+  "bulkPriceInfo": [
+    {
+      "quantity": {
+        "amount": 2,
+        "unit": "Stück"
+      },
+      "price": {
+        "taxType": "GROSS",
+        "formatted": "155,00 €",
+        "amount": 155,
+        "currency": "EUR"
+      },
+      "basePrice": null,
+      "priceSavings": null
+    },
+    {
+      "quantity": {
+        "amount": 3,
+        "unit": "Stück"
+      },
+      "price": {
+        "taxType": "GROSS",
+        "formatted": "144,00 €",
+        "amount": 144,
+        "currency": "EUR"
+      },
+      "basePrice": null,
+      "priceSavings": null
+    }
+  ],
+  "customerSpecificPriceInfo": [
+    {
+      "customerGroup": "KG1",
+      "customerGroupId": "63E4F781-CDA5-327F-B72B-C0A802027724",
+      "priceInfo": [
+        {
+          "quantity": {
+            "amount": 4,
+            "unit": "Stück"
+          },
+          "price": {
+            "taxType": "GROSS",
+            "formatted": "177,60 €",
+            "amount": 177.6,
+            "currency": "EUR"
+          },
+          "basePrice": null,
+          "priceSavings": {
+            "percent": {
+            "formatted": "20 %",
+            "percentage": 20.0
+            },
+            "regularBulkPrice": {
+              "taxType": "GROSS",
+              "formatted": "222,00 €",
+              "amount": 222,
+              "currency": "EUR"
+            }
+          }
+        },
+        {
+          "quantity": {
+            "amount": 9,
+            "unit": "Stück"
+          },
+          "price": {
+            "taxType": "GROSS",
+            "formatted": "90,00 €",
+            "amount": 90,
+            "currency": "EUR"
+          },
+            "basePrice": null,
+            "priceSavings": {
+              "percent": {
+                "formatted": "10 %",
+                "percentage": 10.0
+            },
+            "regularBulkPrice": {
+              "taxType": "GROSS",
+              "formatted": "100,00 €",
+              "amount": 100,
+              "currency": "EUR"
+            }
+          }
+        },
+        {
+          "quantity": {
+            "amount": 55,
+            "unit": "Stück"
+          },
+          "price": {
+            "taxType": "GROSS",
+            "formatted": "70,00 €",
```

Comment:
@mhoehn If the difference between buying 9 instead of 4 pieces result in paying nearly 50 percent less, then in comparison buying 55 instead of 9 should have a way higher impact on the price than 30 percent. I think the prices in this example need to be re-evaluated.